test/math.jl

```diff
@@ -27,6 +27,8 @@ end
         clamp!(x, 1, 3)
         @test x == [1.0, 1.0, 2.0, 3.0, 3.0]
     end
+
+    @test ismissing(clamp(missing, 1, 2))
```

Comment:
Put this in test/missing.jl instead. (I'm not sure putting the definitions in base/math.jl is a good idea, I hadn't noticed they have been moved out of base/missing.jl by https://github.com/JuliaLang/julia/pull/25261.)